gensrc/proto/internal_service.proto

```diff
@@ -322,6 +322,53 @@ message PProxyResult {
     optional PKafkaOffsetBatchProxyResult kafka_offset_batch_result = 102;
 };
 
+message PPulsarLoadInfo {
+    required string service_url = 1;
```

Comment:
all fields in message should be optional

Reply:
This message is pulsar task info, `service_url`, `topic` and `subscription` are required to consume pulsar.

We can also task kafka as reference:

```
message PKafkaLoadInfo {
    required string brokers = 1;
    required string topic = 2;
    repeated PStringPair properties = 3;
};
```